push/src/androidTest/java/org/aerogear/mobile/push/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package org.aerogear.mobile.push;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
```

Comment:
Probably need a better class name.

Reply:
I'm going to kill it since we don't wanna have any instrumented test for now ;)

Reply:
Still needs a kill :)